flutter/shell/platform/common/client_wrapper/include/flutter/event_stream_handler.h

```diff
@@ -13,16 +16,16 @@ class EncodableValue;
 
 template <typename T = EncodableValue>
 struct StreamHandlerError {
-  const std::string& error_code;
-  const std::string& error_message;
```

Comment:
This problem is finally solved!

Reply:
Yes, the workaround you applied to the plugins can now be removed but this change will not break the plugins since there's no plugin that utilizes the `error_details` field.